cmd/statusd/main.go

```diff
@@ -32,10 +32,6 @@ const (
 	serverClientName = "Statusd"
 )
 
-var (
-	buildStamp = "N/A" // rely on linker: -ldflags -X main.buildStamp"
```

Comment:
Why exactly are we dropping this?

Reply:
It's unused. I have never examined this setting and it's preventing from caching more layers when building a docker image.